data/sql_setup/prepare_schedule_a.sql

```diff
@@ -101,13 +48,6 @@ begin
                 delete from ofec_sched_a_queue_new where sub_id = view_row.sub_id;
                 insert into ofec_sched_a_queue_new values (view_row.*, timestamp, two_year_transaction_period);
             end if;
-        else
-            -- We weren't able to successfully retrieve a row from the view,
-            -- so keep track of this sub_id if we haven't already so we can
-            -- try processing it again each night until we're able to
-            -- successfully process it.
-            delete from ofec_sched_a_nightly_retries where sub_id = new.sub_id;
-            insert into ofec_sched_a_nightly_retries values (new.sub_id, 'insert');
         end if;
 
         return new;
```

Comment:
Oh interesting - with this being an `AFTER` trigger the record returned ultimately doesn't matter because it's already been processed in the table that the trigger was fired off of.  I'm hesitant to make the change only because of all of the issues we've had here before with the incorrect return value being set in the past.

Would you be amenable to leaving this as is in this PR, but we can open a separate one to research more once we're certain the nightly update process itself is working fine again?